db/blob/db_blob_basic_test.cc

```diff
@@ -0,0 +1,188 @@
+//  Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "db/blob/blob_index.h"
+#include "db/db_test_util.h"
+#include "port/stack_trace.h"
+#include "test_util/sync_point.h"
+#include "utilities/fault_injection_env.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+class DBBlobBasicTest : public DBTestBase {
+ protected:
+  DBBlobBasicTest()
+      : DBTestBase("/db_blob_basic_test", /* env_do_fsync */ false) {}
+};
+
+TEST_F(DBBlobBasicTest, GetBlob) {
+  Options options;
+  options.enable_blob_files = true;
+  options.min_blob_size = 0;
+  options.disable_auto_compactions = true;
```

Comment:
Maybe add a `TODO` to enable auto compactions?

Reply:
I think the very first test case (which does two `Get`s, the second with `read_tier` set to `kBlockCacheTier`) needs it to be set to ensure the SST and the block are found in the cache. For the rest, I think we could remove it now.

Reply:
Actually, we can remove it from the first test case too: automatic compactions wouldn't kick in with a single L0 file and nothing else.